offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -234,14 +252,34 @@ class OfferToPDFConverter implements OfferExporter {
             String elementId = "product-items" + "-" + tableCount
             htmlContent.getElementById("item-table-grid").append(ItemPrintout.tableHeader(elementId))
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
         } else {
             //otherwise add total pricing to table
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
         }
     }
 
+    void setTaxationStatement() {
+        if(!offer.getSelectedCustomerAffiliation().country.equals(countryWithVAT)) {
+            htmlContent.getElementById("vat-cost-applicable").text("Taxation is not applied to offers outside of ${countryWithVAT}")
+        }
+
+    }
+
+    void setTaxationRatioInSummary() {
+        DecimalFormat decimalFormat = new DecimalFormat("#%")
+        double taxRatio
+        if(offer.getSelectedCustomerAffiliation().getCountry().equals(countryWithVAT) && !offer.getSelectedCustomerAffiliation().getCategory().equals(noVatCategory)) {
```

Comment:
You are using this check multiple times. Maybe it would be an idea to extract it into a small private method (`boolean isTaxApplicable(String country, AffiliationCategory category)`)

Reply:
Good Idea! 💯